test/ad/utils.jl

```diff
@@ -2,7 +2,7 @@
 const AD = get(ENV, "AD", "All")
 
 # Struct of distribution, corresponding parameters, and a sample.
-struct DistSpec{VF<:VariateForm,VS<:ValueSupport,F,T,X,G}
+struct DistSpec{VF <: VariateForm,VS <: ValueSupport,F,T,X,G}
```

Comment:
I think our style-guide says these `<:` should be without the whitespace? Also, there should be whitespace between the type-parameters, e.g. `F,T` should be `F, T`.